features/dd-sdk-android-rum/src/main/kotlin/com/datadog/android/rum/internal/domain/scope/RumViewScope.kt

```diff
@@ -1388,6 +1397,16 @@ internal open class RumViewScope(
             interactionToNextViewMetricResolver: InteractionToNextViewMetricResolver,
             networkSettledResourceIdentifier: InitialResourceIdentifier
         ): RumViewScope {
+            val networkSettledMetricResolver = NetworkSettledMetricResolver(
+                networkSettledResourceIdentifier,
+                sdkCore.internalLogger
+            )
+
+            val viewEndedMetricDispatcher = ViewEndedMetricDispatcher(
+                viewType = ViewMetricDispatcher.ViewType.CUSTOM,
```

Comment:
shouldn't we inherit it as an argument of `fromEvent` call and it is up to the caller to provide a relevant value?

Reply:
I found that there is `RumViewType` enum that does actually the same so removed duplication and reusing  `RumViewType` instead. There is no reason to get the value from method parameters because  `RumViewType` is not provided at this moment, but thanks for spotting - it helped to reduce code duplication